src/koza/model/config/source_config.py

```diff
@@ -147,37 +132,117 @@ class DatasetDescription:
     rights: str = None
 
 
-@dataclass(config=PydanticConfig)
-class SourceConfig:
-    """
-    Base class for primary sources and mapping sources
+@dataclass()
+class SSSOMConfig():
+    """SSSOM config options"""
 
-    TODO document fields
+    files: List[Union[str, Path]] = field(default_factory=list)
+    filter_prefixes: List[str] = field(default_factory=list)
+    target_prefixes: List[str] = field(default_factory=list)
 
-    header: Optional, int|HeaderMode - the index (0 based) in which the
-            header appears in the file.  If header is set to infer
-            the headers will be set to the first line that is not blank
-            or commented with a hash.  If header is set to 'none'
-            then the columns field will be used, or raise a ValueError
-            if columns are not supplied
+    def __post_init_post_parse__(self):
+        self.df = self.merge_and_filter_sssom()
+        self.lut = self.build_sssom_lut()
+
+    def merge_and_filter_sssom(self):
+        mapping_sets = []
+        for file in self.files:
+            msdf = parse_sssom_table(file)
+            mapping_sets.append(msdf)
+        new_msdf = merge_msdf(*mapping_sets)
+        if self.filter_prefixes:
+            new_msdf = filter_prefixes(
+                df=new_msdf.df,
+                filter_prefixes=self.filter_prefixes,
+                require_all_prefixes=False
+            )
+            
+        return new_msdf
+
+    def build_sssom_lut(self) -> Dict:
```

Comment:
This should probably be a private method using the leading `_` in the method name, since it depends on a specific internal state of the class?